config/requirements_development.txt

```diff
@@ -12,4 +12,4 @@ pre-commit
 selenium==3.141.*                           # Functional testing (in here for CI tests)
 
 # Debug
-django-debug-toolbar==2.*
+django-debug-toolbar==3.2.1
```

Comment:
I think we normally wildcard the minor version so that we don't have to update this so often, since it's just a development tool 🙂